spec/support/content_store_helpers.rb

```diff
@@ -0,0 +1,55 @@
+require "govuk_schemas"
+require "gds_api/test_helpers/content_store"
+
+module ContentStoreHelpers
+  include GdsApi::TestHelpers::ContentStore
+
+  def content_store_has_example_item(base_path, schema:, example: nil, is_tagged_to_taxon: false)
+    content_item = GovukSchemas::Example.find(schema, example_name: example || schema)
+
+    content_item["links"] ||= {}
+    content_item["links"]["taxons"] = is_tagged_to_taxon ? [basic_taxon] : []
+    content_item["base_path"] = base_path
+
+    stub_content_store_has_item(base_path, content_item)
+    content_item
+  end
+
+  def basic_taxon
+    {
+      "content_id" => "30c1b93d-2553-47c9-bc3c-fc5b513ecc32",
+      "locale" => "en",
+      "title" => "A Taxon",
+      "base_path" => "/a-taxon",
+    }
+  end
+
+  def content_store_has_random_item(base_path:, schema: "generic", is_tagged_to_taxon: false, details: {})
+    content_item = GovukSchemas::RandomExample.for_schema(frontend_schema: schema) do |item|
+      taxons = is_tagged_to_taxon ? [basic_taxon] : []
+      item.merge!(
+        "base_path" => base_path,
+        "links" => { "taxons" => taxons },
+      )
+      item["details"].merge!(details)
+      item
+    end
+
+    stub_content_store_has_item(content_item["base_path"], content_item)
+    content_item
+  end
+
+  def content_store_has_page(slug, schema: "generic", is_tagged_to_taxon: false)
```

Comment:
I'm not sure why this method is needed. Can the tests not call `content_store_has_random_item` with the specific base_path they require?